test/webdriverio/test/actions_test.ts

```diff
@@ -14,14 +14,12 @@ import {
   tabNavigateToWorkspace,
   testFileLocations,
   testSetup,
+  sendKeyAndWait,
   keyRight,
   contextMenuItems,
 } from './test_setup.js';
 
 suite('Menus test', function () {
-  // Setting timeout to unlimited as these tests take longer time to run
-  this.timeout(0);
```

Comment:
I do wonder about this change as it feels like there's a good chance some longer tests will take longer to run than the default, and this will get really annoying when you're debugging locally... maybe that's not a good enough reason to put unreasonably long timeouts, but I'm a bit hesitant to do something that might make it even more annoying to write tests than it already is.

Reply:
Hmm.  It is true that the default timeout does make debugging tests annoying.  Maybe we should set the timeout to zero iff `PAUSE_TIME` is nonzero?  Or set it to a suitable multiple of `PAUSE_TIME`?

Reply:
Setting the timeout if pause time is nonzero seems like a good idea. If for some reason that isn't possible then I would probably lean towards having the infinite timeout to make it easier to write tests personally.